docs/latest/design/bootstrap.md

```diff
@@ -35,7 +35,7 @@ type EnvoyProxySpec struct {
 	// from it are not configurable and will result in the `EnvoyProxy` resource being rejected.
 	// We strongly recommend using `egctl x translate` to generate a `EnvoyProxy` resource with the `Bootstrap` field set to the default
 	// Bootstrap configuration used. You can edit this configuration, and rerun `egctl x translate` to ensure there are no validation errors.
-	Bootstrap string `json:"bootstrap,omitempty"`
+	Bootstrap *string `json:"bootstrap,omitempty"`
```

Comment:
why need a point here? Is there difference between nil and empty string?

Reply:
yes, nil is a better check than empty string. This is just updating docs, this change was already made to the API

Reply:
Can you elaborate?

Reply:
yes thats right

Reply:
so what's the difference between nil and empty string？

why nil is better？

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required recommends pointer type for optional fields

Reply:
make sense